test/e2e_node/mirror_pod_test.go

```diff
@@ -370,6 +372,20 @@ func createStaticPodUsingNfs(nfsIP string, nodeName string, cmd string, dir stri
 	return nil
 }
 
+// NFSv3 and v4 clients mount exported paths differently. This is an attempt to try to gloss over
+// those differences and make this test reliable for both versions.
+func mountPath(path string) string {
```

Comment:
I wrote it this way so it could be reused if there are other test failures for this same reason, but this seems to be the only one for now. Happy to change it.